test/lib/unit/utils/lb_spec.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+
+const assert = require("assert");
+const sinon = require("sinon");
+const path = require("path");
+const LbHelper = require("../../../../lib/utils/lb");
+const LabShareConfig = require("../../../../lib/utils/labshare-config");
+const testModulePath = path.join(__dirname ,"../../../fixtures/package")
+
+
+describe('lb Utils', () => {
+
+    afterEach(() => {
+            sinon.restore();
+          
+        });
+
+    it('lb getApiRestSettings with defined config', async () => {
```

Comment:
Could you use the active voice/action description for the tests instead? For example: `it('loads the rest API settings'`).